_posts/fork/2021-02-04-Scrolls.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: theme
+title: scrolls
+category: theme
+date: 2021-02-04
+homepage: https://github.com/daodaolee/typora-scrolls
+download: https://github.com/daodaolee/typora-scrolls/archive/0.2.zip
+built-in: false
+author: DaoDaoLee
+thumbnail: thumbnail
```

Comment:
thumbnail should be scrolls.png

Reply:
i got it, so do I need to resubmit no? because I see that the gh-page branch does not have my theme, but the scrolls branch of daodaolee has